.github/chainguard/ShopwareAcademy.sts.yaml

```
@@ -0,0 +1,14 @@
+# Allows Shopware Learning Environments in Community Hub be pre-installed with selected premium extensions
+
+issuer: https://token.actions.githubusercontent.com
+subject_pattern: repo:shopware/academy-shopdev-plugin:.*
+
```

```suggestion
claim_pattern:
  workflow_ref: shopware/academy-shopdev-plugin/.github/workflows/create-template.yml@.*
```
This limits the token to the `create-template.yml` workflow :slightly_smiling_face: